services/frontend-service/src/ui/components/ReleaseDialog/ReleaseDialog.scss

```diff
@@ -2,60 +2,163 @@
+  background: var(--mdc-theme-primary);
+  color: var(--mdc-theme-on-primary);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
 }
 
 .release-dialog-app-bar-data {
-    margin: 24px 10px 24px 47px;
-    display: flex;
-    flex-direction: column;
-    justify-content: space-between;
+  margin: 24px 10px 24px 47px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
 }
 
 .release-dialog-message {
-    @extend .sub-headline1;
-    display: flex;
-    justify-content: space-between;
+  @extend .sub-headline1;
+  display: flex;
+  justify-content: space-between;
 }
 
 .release-dialog-commitId {
-    height: 28px;
-    white-space: nowrap;
-    width: auto;
-    padding: 2px 6px;
-    border: 1px solid var(--mdc-theme-on-primary);
-    border-radius: 10px;
-    margin-top: 26px;
-    margin-bottom: auto;
-    text-align: center;
-    @extend .text-regular;
+  height: 28px;
+  white-space: nowrap;
+  width: auto;
+  padding: 2px 6px;
+  border: 1px solid var(--mdc-theme-on-primary);
+  border-radius: 10px;
+  margin-top: 26px;
+  margin-bottom: auto;
+  text-align: center;
+  @extend .text-regular;
 }
 
 .release-dialog-commitMessage {
-    text-overflow: wrap;
+  text-overflow: wrap;
 }
 
 button.release-dialog-close {
+  display: flex;
+  margin: 20px 10px 100px;
+}
+
+.release-env-list {
+  background-color: var(--mdc-theme-surface);
+  display: flex;
+  flex-direction: column;
+  overflow: overlay;
+  gap: 22px;
+  margin: 0px;
+  padding: 35px 0px;
+  align-items: center;
+  border-radius: 0px 0px 15px 15px;
+}
+
+.env-card {
+  background-color: var(--mdc-theme-background);
+  display: flex;
+  flex-direction: column;
+  width: 900px;
+  height: 160px;
+  padding: 0px;
+  margin: 0px 20px;
+  border-radius: 10px;
+  gap: 16px;
+}
+
+.env-card-header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.env-card-label {
+  border-radius: 10px;
+  width: auto;
+  height: 42px;
+  padding: 10px;
+  background-color: #FF8A00;
+  color: var(--mdc-theme-on-primary);
+  @extend .text-bold;
+  position: relative;
+  top: -10px;
+  left: -10px;
+  display: flex;
+  flex-direction: row;
+  .env-card-env-locks:nth-child(2) {
+    margin-left: 55px;
     display: flex;
-    margin: 20px 10px 100px;
+    flex-direction: row;
+  }
+}
+
+svg.env-card-env-lock-icon {
+  width: 24px;
+  height: 24px;
+  margin: unset;
+}
+
+button.button-lock {
+  min-width: unset;
+  padding: unset;
+  height: auto;
+}
+
+.env-card-data {
+  display: flex;
+  flex-direction: row;
+  padding-left: 27px;
+  @extend .text-medium;
+}
+
+.env-card-app-locks {
+  display: flex;
+  position: relative;
+  top: 20px;
+  right: 36px;
+  flex-direction: row;
+  gap: 13px;
+}
+
+.env-card-buttons {
+  margin-left: auto;
+  margin-right: 36px;
+  margin-bottom: 17px;
+  display: flex;
+  flex-direction: row;
+  gap: 20px;
+  button {
+    padding: 8px 19px;
+  }
+}
+
```

Comment:
overall there are a lot of css rules here, are these all necessary?
Please try to remove some.